xarray/core/duck_array_ops.py

```diff
@@ -229,26 +243,19 @@ def f(values, axis=None, skipna=None, **kwargs):
     return f
 
 
-argmax = _create_nan_agg_method('argmax', coerce_strings=True,
-                                support_object_type=True)
-argmin = _create_nan_agg_method('argmin', coerce_strings=True,
-                                support_object_type=True)
+argmax = _create_nan_agg_method('argmax', coerce_strings=True)
+argmin = _create_nan_agg_method('argmin', coerce_strings=True)
 max = _create_nan_agg_method('max', coerce_strings=True,
                              support_object_type=True)
 min = _create_nan_agg_method('min', coerce_strings=True,
                              support_object_type=True)
 sum = _create_nan_agg_method('sum', numeric_only=True,
                              support_object_type=True)
```

Comment:
Just a workaround for `.sum` to work with object-dtype.
I tried to generalize this to other ops, but it was not very easy because dask, bottleneck, and numpy work differently.
I will raise another issue for this but I want to finish this up with the smallest workaround.